amundsen_application/static/css/_typography-custom.scss

```diff
@@ -1 +1,5 @@
-// This file is intentionally left blank and should be overwritten by in the build process.
+.truncated {
```

Comment:
This file should remain untouched. I'm not sure where the best place for `truncated` is, at best probably `_typography-default` since this applies to our text and is part of the default built in styling.